ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml

```diff
@@ -0,0 +1,149 @@
+---
+- name: Test cv_container_v3 for configlet
+  hosts: CloudVision
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.cvp
+  vars:
+
+    ansible_command_timeout: 1200
+    ansible_connect_timeout: 600
+
+    TOPOLOGY_3_TIER:
+      TEAM01:
+        parentContainerName: Tenant
+      TEAM01_DC:
+        parentContainerName: TEAM01
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+        configlets:
+          - ATD-INFRA
```

Comment:
can we use our own configlet instead? I don't want to rely on ATD configlets

Reply:
Yes, we can use any configlet, whichever is available on your lab.

Reply:
Fixed!